src/General/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { Alert, Props } from './Alert';
+import { Button } from '../Button';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+
+export default {
+  title: 'General/Alert',
+  component: Alert,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+  argTypes: {
+    type: {
+      control: {
+        type: 'select',
+        options: ['success', 'warning', 'danger', 'info'],
+      },
+    },
+  },
+} as Meta;
```

Comment:
Can disable the control func for className
![image](https://user-images.githubusercontent.com/13249748/101117588-3e7b7b80-3622-11eb-9d0b-2d69abeb9332.png)